KomaMRICore/src/simulation/SimulatorCore.jl

```diff
@@ -185,8 +257,9 @@ function simulate(
 )
     #Simulation parameter unpacking, and setting defaults if key is not defined
     sim_params = default_sim_params(sim_params)
+    sampling_params = Dict{String,Any}("Δt"=>sim_params["Δt"], "Δt_rf"=>sim_params["Δt_rf"])
```

Comment:
```suggestion
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/241/commits/72790d5e927932b0a83b571459e14a4602156038